donkeycar/parts/augumentations.py

```diff
@@ -0,0 +1,73 @@
+import cv2
+import numpy as np
+
+
+class Augumentations(object):
+    """
+    Some ready to use image augumentations.
+    """
+
+    @classmethod
+    def crop(cls, left, right, top, bottom, keep_size=False):
+        '''
+        The image augumentation sequence.
+        Crops based on a region of interest among other things.
+        left, right, top & bottom are the number of pixels to crop.
+        '''
+        import imgaug as ia
+        import imgaug.augmenters as iaa
+
+        augmentation = iaa.Sequential([
+            iaa.Crop(
+                px=(top, right, bottom, left),
+                keep_size=keep_size
+            ),
+        ])
+        return augmentation
+
+    @classmethod
+    def trapezoidal_mask(cls, lower_left, lower_right, upper_left, upper_right, min_y, max_y):
```

Comment:
Nice!  When does this get applied in training and autopilot.  More specifically; I want to know if this alters data on the way into the tub, or just at training and inference time.